manifests/install.pp

```diff
@@ -11,7 +11,7 @@
 
   exec {
     'Download Grafana':
-      command => "curl -s -L http://grafanarel.s3.amazonaws.com/grafana-${::grafana::version}.tar.gz | tar xz",
+      command => "curl -s -L ${::grafana:package_base}/grafana-${::grafana::version}.tar.gz | tar xz",
```

Comment:
As the CI mentions, there is a syntax error. Or better be said: there should be two ::